src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days < 7) {
+    totalCost = totalCost - 20;
+
+    return totalCost;
+  } else {
+    totalCost = totalCost - 50;
+
+    return totalCost;
```

Comment:
Avoid using literals like `20` and `50` directly in the expressions. Instead, define them as constants with descriptive names to improve code readability and maintainability, as per the checklist item 2.